src/backends/tensorrt/plan_backend.cc

```diff
@@ -280,7 +280,7 @@ PlanBackend::CreateExecutionContexts(
 
   // Only need to map device to runner when creating contexts, after that,
   // only runner idx is needed.
-  std::map<int, size_t> device_to_runner_map;
+  std::map<std::pair<int, int64_t>, size_t> device_to_runner_map;
```

Comment:
Can you add a comment here about using the pair? It will help me when making changes to the TRT backend repo. Basically a pair of primary device and secondary device will compose the **"device"** which will have their own context.

Reply:
Done, marked as WIP for now till other iGPU issues are resolved.